api/peripheral_api/routes/Printer.js

```diff
@@ -73,19 +81,32 @@ router.post('/sendPrintRequest', async (req, res) => {
 
   const accountId = ACCOUNT_ID;
   const queueName = PAPER_PRINTING_QUEUE_NAME;
-
-  const sqsParams = {
-    MessageBody: JSON.stringify({
-      location: response.Location,
-      fileNo: fileName,
-      copies: copies,
-      pageRanges,
-    }),
-    QueueUrl: `https://sqs.us-west-2.amazonaws.com/${accountId}/${queueName}`
-  };
-  sqs.sendMessage(sqsParams, (err, data) => {
-    return res.sendStatus(OK);
+  const data = JSON.stringify({
+    location: response.Location,
+    fileNo: fileName,
+    copies: copies,
+    pageRanges,
   });
+  const result  = SqsHandler.pushMessageToQueue(data);
+
+  if(!result){
+    return res.sendStatus(BAD_REQUEST);
+  }
+  return res.sendStatus(OK);
+});
+
+router.post('/pushDiscordPDFToSqs', async (req, res) => {
+  const { apiKey, fileURL } = req.body;
+  if(!checkDiscordKey(apiKey)){
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  const result = await SqsHandler.pushMessageToQueue({fileURL});
```

Comment:
can we make this more descriptive? like
```suggestion
  const result = await SqsHandler.pushMessageToQueue({discordFileUrl});
```

that way on the quasar side we can do
```js
if (sqsMessage.discordFileUrl) {
  // handle special discord logic
} else {
  // default quasar logic
}
```